flight_planning/v1/telemetry.yaml

```diff
@@ -0,0 +1,220 @@
+info:
+  title: >-
+    Telemetry definitions used in flight planning interface.
+components:
+  schemas:
+    FlightTelemetry:
+      description: >-
+        When this information is present and the USS supports telemetry ingestion for a flight, the USS receiving this information should enqueue these telemetry reports to be delivered to the system, as if they were coming from the user, at (or soon after) the specified times.  If/when the associated flight is canceled or closed, any telemetry not yet delivered to the system should be dequeued and discarded.
+      type: object
+      properties:
+        states:
+          description: >-
+            The set of telemetry data that should be injected into the system (as if reported by the user or the user's system) at the appropriate times (and not before) for this flight.
+          type: array
+          items:
+            $ref: '#/components/schemas/AircraftState'
+    AircraftState:
+      description: State of an aircraft for the purposes of simulating the execution of a flight plan.
+      required:
+        - timestamp
+      type: object
+      properties:
+        id:
+          description: >-
+            Unique identifier for this aircraft state/telemetry.  If a USS has already queued an aircraft state with this ID as telemetry to be delivered, this state may be ignored.
+          type: string
+        timestamp:
+          description: >-
+            Time at which this state is valid.  This is equivalent to the time of sensor measurement, so the USS's primary system under test should not be aware of this state until after this time.
+          anyOf:
+            - $ref: 'geotemporal.yaml#/components/schemas/Time'
+        timestamp_accuracy:
+          description: >-
+            Declaration of timestamp accuracy, which is the one-sided width of the 95%-confidence interval around `timestamp` for the true time of applicability for any of the data fields.
+          type: number
+          format: float
+          minimum: 0
+          example: 1
+          default: 0
```

Comment:
I would suggest making the min, example, and default be more "floaty" for clarity.  0.0, 1.2, 0.0 for example.

Reply:
Sure; sounds good.